.gitignore

```diff
@@ -86,3 +86,4 @@ quarkus/data/*.db
 
 # Node.js for frontend-maven-plugin #
 node
+node_modules
```

Comment:
I don't think we need this as it is already in the .gitignore file for the js directory and themes directory.  That should be enough.

Reply:
Fixed